src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/GroupFlowPathModel.groovy

```diff
@@ -0,0 +1,13 @@
+package org.openkilda.functionaltests.helpers.model
```

Comment:
Do we use group flow paths somewhere in tests? If no, I guess we don't need to create classes and deserialize it.

Reply:
sure, let's keep it simple